include/rocksdb/options.h

```diff
@@ -1361,6 +1361,15 @@ struct IngestExternalFileOptions {
   bool verify_checksums_before_ingest = false;
 };
 
+enum TraceFilterType: unsigned char {
```

Comment:
Lets change the enum be of type `uint64_t` as a char might not be sufficient if we have filter types more than 8, based on how we are using it (one bit per type).